### PR TITLE
[10.0][WIP][post_list_order] Make possible to list pos orders in the POS (in this moment only available in backend)

### PR DESCRIPTION
**Pull request by ovnicraft**
Add a button to list all orders for active session
* [ ] include state on list
* [ ] search

Please give your feedback about.
button un main screen
captura de pantalla 2017-02-10 a las 12 22 53

List orders

captura de pantalla 2017-02-10 a las 12 23 02

### CODE REVIEW

**Comment by legalsylvain** [comment]
Hi @ovnicraft. LGTM. (not tested, because runbot failed). I don't know why.

Just on thing : I think that the name of your module is not very clear. pos_list_order, should be better I think. (module name are in the singular by convention, in OCA)

I'm not sure to see the use case for the end user. (I mean, it's seems that you have the same information in the back office, with the same usability, don't you ?)
Or maybe did you planned to do after, some missing feature like directly in the Front UI : 
- return order
- print again ticket
- merge feature present in module pos_order_load module (PR V8)

Thanks.

**Comment by ovnicraft** [comment]
@legalsylvain yes, in fact i am working to reprint ticket order and add a total for session, looks like fast report to cashier. 
So i will rename the module in favor `post_list_order`
I used in runbot, works fine i dont know why failed.

File: pos_orders/views/templates.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<odoo>
```
Comment (elicoidal) [inline pos_orders/views/templates.xml:1]: remove file

File: pos_orders/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
```
Comment (elicoidal) [inline pos_orders/__manifest__.py:1]: @ovnicraft   Thanks for your contribution to the OCA. 
Let me introduce to you some basic information about contribution in OCA and I will review other points in more comments inside the code..
In general, please follow the principles: 
* Contribution guidelines: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md
* Please follow the OCA README template:
https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst
* We recommend to include a screenshot as well to help new comers (nice to have)
* Please use simple headers in all py files following OCA templates
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3

File: pos_orders/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+{
+    'name': "List POS Orders",
+    'summary': """
+    Add option in POS UI to list and print generated orders """,
+    'author': "Cristian Salamea",
+    'website': "http://www.ayni.com.ec",
+    'category': 'Point Of Sale',
+    'version': '10.0.0.0.1',
```
Comment (elicoidal) [inline pos_orders/__manifest__.py:9]: 10.0.1.0.0

File: pos_orders/static/src/js/pos.js [hunk 4]
```diff
@@ -0,0 +1,157 @@
+odoo.define('pos_orders.pos', function (require) {
```
Comment (elicoidal) [inline pos_orders/static/src/js/pos.js:1]: add copyright

**Review by elicoidal — CHANGES_REQUESTED** [review]
some formal details.
Not tested

File: pos_orders/__manifest__.py (hunk 2 above)
Comment (ovnicraft) [reply pos_orders/__manifest__.py:1]: Ok will apply changes and update PR

**Comment by legalsylvain** [comment]
hi @ovnicraft, any news about that PR, regarding remarks ?

**Comment by rafaelbn** [comment]
Hello @ovnicraft , are you going to continue this PR. I'm interested in testing it. In this moment rebuilding runbot for testing :smile: thanks!

**Comment by rafaelbn** [comment]
Hi again @ovnicraft maybe you would like to review https://github.com/OCA/pos/pull/269 :smile:

**Comment by rafaelbn** [comment]
Hi @ovnicraft I have reviewed (runbot have a warning but works) and it's great! :smile: 

My test was:

- Create 3 order
- Try to list them (this doesn't work)
- Close POS
- I checked that in the backend there was that 3 ordersin backend and they were
- I resume the POS and tried to list de order and NOW YES they are
- I created a new 3 orders (total 6) and list them, NOT I can only see the 3 first order but no the recent ones

So I looks good but still missing some things 

Do you have more improvements. I will test and review you PR. Thanks!

**Comment by rafaelbn** [comment]
Resume:

- Need fixing because there are comments fo fix
- WIP because the work is still not finished
- Need review because it works, is great and more people could be interested

**Comment by legalsylvain** [comment]
Hi. You could be interested by the implementation of this module that makes the same things in V8. 
https://github.com/grap/grap-odoo-incubator/tree/8.0/pos_done_order_load (with reprint function)

regards.

File: pos_orders/static/src/js/pos.js [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+
+
+      this.$('.back').click(function(){
+        self.gui.back();
+      });
+
+      // add print function
+
+      if (self.orders.length == 0){
+        this.load_orders();
```
Comment (legalsylvain) [inline pos_orders/static/src/js/pos.js:43]: I think you should allways reload orders.

**Comment by ovnicraft** [comment]
Hi, i cannot start until 2 weeks, i am busy by now, but i will fix all requests.

Thanks for your comments and reviews, i will take in care all of them.

**Comment by legalsylvain** [comment]
Hi @ovnicraft. No now from one year. I'm closing this PR.
I think that since your work, other module has appeared in V11 / V12 that make the job. pos_order_mgmt. I think that it could be more easy to backport from 11.0 to 10.0 this module.

kind regards.

**Pull request closed without merging**